src/SchemaExplorer.tsx

```diff
@@ -416,29 +418,19 @@ export type SchemaExplorerProps = {
   schema: JsonSchema1;
   stage: Stage;
   lookup: Lookup;
+  onSelectValidationRange: (range: IRange) => void;
+  validationResults: editor.IMarker[]
 };
 
-export type ViewType = 'details' | 'example-json' | 'example-yaml';
+export type ViewType = 'details' | 'example-json' | 'example-yaml' | 'validator';
```

Comment:
TBH, adding this as a new tab is weird because it contains global results. Longer term, it would have been better for this validation view to be provided externally to this tabbed component but, for now, I'll allow it.